apps/condo/domains/ticket/components/TicketForm/CreateTicketForm.tsx

```diff
@@ -27,12 +27,16 @@ const DEFAULT_TICKET_SOURCE_CALL_ID = '779d7bb6-b194-4d2c-a967-1f7321b2787f'
 export const CreateTicketActionBar = ({ handleSave, isLoading }) => {
     const intl = useIntl()
     const CreateTicketMessage = intl.formatMessage({ id: 'CreateTicket' })
+    const AddressNotSelected = intl.formatMessage({ id: 'field.Property.nonSelectedError' })
 
     return (
         <Form.Item noStyle shouldUpdate>
             {
-                ({ getFieldsValue }) => {
+                ({ getFieldsValue, getFieldError }) => {
                     const { property, details, placeClassifier, categoryClassifier, deadline } = getFieldsValue(REQUIRED_TICKET_FIELDS)
+                    const propertyMismatchError = getFieldError('property').map(error => {
```

Comment:
this is strange. this would return nothing is error is not on 0-th index.

Reply:
There can be 10 errors in the array. I `map` it with a condition
`if (error.includes(AddressNotSelected)) return error`
so the array can only have this error. `map()` seems to just `push` to a new array and so if it's not empty then the error will always be at index 0

Reply:
this won't work as decided... :(

Reply:
let's fix this.